### DISCUSSION TITLE
cmp completion is now duplicated

### DISCUSSION THREAD

**Discussion by yriveiro** [folke/noice.nvim#176] [Q&A] [answered]
Hi,

Since the latest update of `noice` I'm getting autocompletion suggestions in duplicate. Is anyone getting the same behaviour?

This is my current configuration of the `lsp` section:

```lua
  lsp = {
    override = {
      -- override the default lsp markdown formatter with Noice
      ["vim.lsp.util.convert_input_to_markdown_lines"] = true,
      -- override the lsp markdown formatter with Noice
      ["vim.lsp.util.stylize_markdown"] = true,
      -- override cmp documentation with Noice (needs the other options to work)
      ["cmp.entry.get_documentation"] = true,
    },
  }
```

<img width="1159" alt="Screenshot 2022-10-29 at 18 15 38" src="https://user-images.githubusercontent.com/934763/198844760-39471057-6de5-48ac-9c52-ed0888db527b.png">

**Comment by folke** [comment, marked as answer]
One is Noice signature help, the other is cmp signature help. You should disable one of the two

**Reply by yriveiro** [reply]
I disabled `nvim_lsp_signature_help`from the sources of cmp and worked. A silly question, can I uninstall `hrsh7th/cmp-nvim-lsp-signature-help` plugin or is still required by noice to get the signature information?

**Reply by folke** [reply]
Nope, you can uninstall it

**Reply by nyngwang** [reply]
@yriveiro I just encountered this problem, thanks for your asking! With treesitter the docs look much better!

**Reply by folke** [reply]
@nyngwang thanks!

**Reply by samEngineering** [reply]
@yriveiro,
Can you share how you disabled that in your cmp config plz? I am facing the same issue, but I do not have `hrsh7th/cmp-nvim-lsp-signature-help`, nor do I have `nvim_lsp_signature_help` listed in my `cmp` sources. I only have the following:

```
sources = {
   { name = "buffer" },
   { name = "nvim_lsp" },
   { name = "luasnip" },
},
```

**Reply by yriveiro** [reply]
Hi, @samEngineering, I followed Folke's instructions, uninstalled the source and the problem disappeared.
